third_party/googletest/src/googletest/src/gtest-port.cc

```diff
@@ -1026,7 +1028,7 @@ GTestLog::GTestLog(GTestLogSeverity severity, const char* file, int line)
 GTestLog::~GTestLog() {
   GetStream() << ::std::endl;
   if (severity_ == GTEST_FATAL) {
-    fflush(stderr);
+    posix::Flush();
```

Comment:
Same comment as above for malloc/free: Maybe a  `#define fflush SbLogFlush` and `#define printf SbLogFormat` at the top and undef again at the end would be cleaner ? Again note, `fflush` and `printf` aren't posix, they are standard C